pages/Farmer-Androids.md

```diff
@@ -0,0 +1,17 @@
+An Android that specializes in automating the farming process.
+This android only works with planted seeds and will not work with melons or pumpkins.
```

Comment:
or sugar cane, or cactus.
We should probably add a table here with the seeds that **do** work.